linera-chain/src/manager/mod.rs

```diff
@@ -79,12 +87,9 @@ impl ChainManager {
         }
     }
 
-    pub fn next_round(&self) -> RoundNumber {
+    pub fn current_round(&self) -> RoundNumber {
```

Comment:
What is the deal between `next_round` and `current_round` again?

Reply:
I think I should actually remove `next_round`, and move that logic to the client side.

The difference is that `next_round` can sometimes be `None` or the successor of `current_round`:
`current_round` is the round that the timeout is currently running for; the one where the validators could still vote for `ValidatedBlock` or `ConfirmedBlock`.
The `next_round` is the one where you should propose a block right now, if you want to. That could be the current one, if no block has been proposed yet, or the next round, if we're in multi-leader mode, nor `None`, if we're in single-leader mode and a block has already been proposed in the current round.

Reply:
Done.